dell/anomaly_alert.py.txt

```diff
@@ -0,0 +1,133 @@
+import requests
+import time
+import json
+# import pendulum
+import pickle
+import logging
+import configparser
+from dateutil.tz import gettz
+from datetime import datetime, timedelta
+
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+def log_in(host, user_name, password, user_agent):
```

Comment:
remove the unnecessary code

Reply:
this has been removed in anomaly_alert.py. will remove the txt file.